tests/build-newsroom-videos.test.js

```diff
@@ -0,0 +1,62 @@
+const { buildNewsroomVideos } = require('../scripts/build-newsroom-videos');
+const fetch = require('node-fetch');
+const { resolve } = require('path');
+const { mkdirSync, readFileSync, rmSync } = require('fs');
+const { mockApiResponse, expectedResult } = require('./fixtures/newsroomData');
+
+const testDir = resolve(__dirname, 'test_config');
+const testFilePath = resolve(testDir, 'newsroom_videos.json');
+
+jest.mock('node-fetch', () => jest.fn());
```

Comment:
need to see will this `node-fetch` work or not

Reply:
The test is working because we are not actually calling it, but instead we are mocking it.